doc/api/modules.md

```diff
@@ -61,7 +61,36 @@ module.exports = class Square {
 };
 ```
 
-The module system is implemented in the `require('module')` module.
+The CommonJS module system is implemented in the [`module` core module][].
+
+## Enabling
+
+<!-- type=misc -->
+
+Node.js has two module systems: CommonJS modules and [ECMAScript modules][].
+
+By default, Node.js will treat the following as CommonJS modules:
+
+* Files with a `.cjs` extension;
+
+* Files with a `.js` extension when the nearest parent `package.json` file
+  contains a top-level field [`"type"`][] with a value of `"commonjs"`.
+
+* Files with a `.js` extension when the nearest parent `package.json` file
+  doesn't contain a top-level field [`"type"`][]. Package authors should 
+  include the [`"type"`][] field, even in packages where all sources are
+  CommonJS, in case the default ever changes.
```

Comment:
I don’t think this is reasonable advise; the default can in practice *never* change, since it would break far too much of the ecosystem. If the ecosystem adopts the field en masse, then that’d be one thing, but i don’t see any evidence that kind of adoption will ever happen for anything.